packages/core/src/components/step-by-step/step-by-step.stories.js

```diff
@@ -0,0 +1,76 @@
+import './step-by-step.css';
+
+export default {
+  title: 'Components/Step By Step',
+};
+
+export const NumberedStepByStep = () => `
+  <div class="hds-step-by-step__container">
+    <header>
+      <h2>Numeroitu vaiheistuskomponentti</h2>
+      <section>Numeroitu prosessi soveltuu hyvin tapauksiin, joissa vaiheiden järjestys on selkeä.</section>
```

Comment:
This is a bit unorthodox way of using `<section>`. 🤔  I would suggest `<div>` or `<span>`

Reply:
I kind of wish I had some kind of an explanation for this when I initially wrote this. No idea why I had used the section component here 😬😬😬